stingray/tests/test_events.py

```diff
@@ -336,19 +338,20 @@ def test_fits_with_standard_file(self):
         """
         fname = os.path.join(datadir, 'monol_testA.evt')
         ev = EventList()
-        ev = ev.read(fname, format_='fits')
-
-    def test_io_with_wrong_format(self):
-        """Test that io methods raise Key Error when
-        wrong format is provided.
-        """
-        ev = EventList()
-        with warnings.catch_warnings(record=True):
-            with pytest.raises(KeyError):
-                ev.write('ev.pickle', format_="unsupported")
-
-        with pytest.raises(KeyError):
-            ev.read('ev.pickle', format_="unsupported")
+        ev = ev.read(fname, format_='hea')
+        assert np.isclose(ev.mjdref, 55197.00076601852)
+
+    # def test_io_with_wrong_format(self):
+    #     """Test that io methods raise Key Error when
+    #     wrong format is provided.
+    #     """
+    #     ev = EventList()
+    #     with warnings.catch_warnings(record=True):
+    #         with pytest.raises(KeyError):
+    #             ev.write('ev.pickle', format_="unsupported")
+    #
+    #     with pytest.raises(KeyError):
+    #         ev.read('ev.pickle', format_="unsupported")
```

Comment:
Is this supposed to be commented out? Should this test be run or deleted?

Reply:
Deleted.